install-sw.yml

```diff
@@ -47,6 +47,20 @@
        - cluster_type in ("NONE", "DG")
   tags: rdbms-setup
 
+
+- hosts: dbasm
+  serial: 1
```

Comment:
Why serial?  Can this be expressed as a dependency instead?

Reply:
Removed this `serial:1` clause and tested successfully against 2 BMS hosts.